tests/php/bootstrap.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * This is a wrapper for the actual bootstrapper. The original one determines the plugin to require
+ * by looking in the root folder and finding the first PHP file that has a WordPress Header block
+ * comment, which unfortunately will be `amp-beta-tester.php`.
+ *
+ * Thankfully (for us), it keeps track of the plugin file to require in a global variable called
+ * `$_plugin_file`. One solution (hack) to fix this is to change the current directory to one where
+ * no plugin files can be found, then provide the path to the actual plugin file to require through
+ * the aforementioned global variable :D .
+ */
+
+global $_plugin_file;
+$current_dir = __DIR__;
+
+// Path to plugin to require.
+$_plugin_file = $current_dir . '/../../amp.php';
```

Comment:
Could we extend the `WP_TEST_ACTIVATED_PLUGINS` environment variable with the additional plugin file instead?

https://github.com/ampproject/amp-wp/blob/e9aa2f3ebc2080106c8d54a07fe4a75f1df84184/phpunit.xml#L10

Reply:
My understanding is that this is a workaround to ensure `amp.php` is loaded for the test framework, whereas `WP_TEST_ACTIVATED_PLUGINS` is for loading additional plugins that are required by the tests.

Without this workaround, the test framework tries to bootstrap by wrongly loading `amp-beta-tester.php` (as it's the first plugin file it finds)

Reply:
Do we want to include `amp/amp-beta-tester.php` in `WP_TEST_ACTIVATED_PLUGINS` if the main plugin file is set to `amp.php`?